azure-pipelines.yml

```diff
@@ -53,15 +53,14 @@ jobs:
         - python-devel
 
     envs:
-      - macos: py36
-        name: py36_test
-        posargs: -n=1
+      - macos: py38
+        name: py38_test
 
       - windows: py37
         name: py37_test
 
-      - linux: py38-online
-        name: py38_test_online
+      - linux: py39-online
+        name: py39_test_online
```

Comment:
Adding python 3.9 support should get a changelog entry.

Reply:
Sure.